libs/ui/popover/.storybook/main.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  addons: ['@storybook/addon-knobs/register']
```

Comment:
I wonder if we can extend a base config like eslint/etc for things like this that we know everyone will need..

Reply:
Yeah, will open a separate PR for all chromatic related tweaks..